src/toil/cwl/cwltoil.py

```diff
@@ -59,6 +59,10 @@
 
 cwllogger = logging.getLogger("cwltool")
 
+# Define internal jobs we should avoid submitting to batch systems and logging
+CWL_INTERNAL_JOBS = ("CWLJob", "CWLJobWrapper", "CWLWorkflow", "CWLScatter", "CWLGather",
```

Comment:
Isn't `CWLJob` the actual running of the job itself and not just a synthetic job?

Reply:
For batch systems CWLJob dispatches the actual job, so it doesn't get labelled as this step. So, the jobs themselves do still go the batch queue. Thanks for reviewing this.